Changelog.md

```diff
@@ -2,13 +2,11 @@
 
 ## 0.0.1
 
-- added support for ConfigMap and Secret backed environment variables
-- added ability to create / modify / delete environment variables
-- added support for ConfigMap and Secret backed datasource properties
 - completion to create a Database view in *.ddl files
-- added ability to create / modify / delete data source properties
-- added ability to create / delete data sources
-- added ability to modify the DDL part of the VDB in an editor
+- added support for ConfigMap / Secret (for both environment variables and datasource properties)
+- added support for environment variables
+- added support for data sources and datasource properties
+- added support for modifying the DDL part of the VDB in an editor
 - added extension specific output channel for logging
 - added Data Virtualization tree view to visualize VDB content
 - ability to create a VDB definition file with basic DDL
```

Comment:
optional: What about placing these two entries at the top. I think they are the 2 most important ones

Reply:
Is the ordering of change log items really an important issue? Just wondering as it is just a list of things that have been done for a given release. I am rather unsure if we really ordered those items by importance in the other vscode extensions. I have no strong opinion, am just curious.

Reply:
I have always left them in the order in which the changes occurred, but your mileage may vary.

Reply:
I marked it as optional.

I don't think it is important, just better.
And usually i try to sort by importance when I add one yes.